### ISSUE TITLE
Cosmic ashwalker shields can be disarmed, somehow.

### ISSUE THREAD

**Issue by LarissaMayrink** [yogstation13/yogstation#2384]
Cosmic ashwalker shields can be disarmed and they are supposed to work just like the riot shield

**Comment by Super3222** [comment]
I think Stealth made it a version of the prototype shield which isn't used for anything.

**Comment by ShadowDeath6** [comment]
don't shields not protect from the back side or was that some fever dream I had

**Comment by Super3222** [comment]
> don't shields not protect from the back side or was that some fever dream I had

yeah, they don't protect from the backside. it was one of the first major projects i did. maybe i'm wrong, but i still feel like the cosmic ashwalkers shield is wonky.
